util/src/builder/command.rs

```diff
@@ -56,13 +54,15 @@ impl CommandBuilder {
 
     /// Consume the builder, returning a [`Command`].
     #[allow(clippy::missing_const_for_fn)]
+    #[must_use = "must be built into a command"]
     pub fn build(self) -> Command {
         self.0
     }
 
     /// Set the application ID of the command.
     ///
     /// Defaults to [`None`].
+    #[must_use]
```

Comment:
I believe `#[must_use]` on the struct achieves the same thing but with less code